flang/lib/Semantics/resolve-names.cpp

```diff
@@ -3627,6 +3627,27 @@ void ModuleVisitor::Post(const parser::UseStmt &x) {
       }
     }
   }
+  // Go through the list of COMMON block symbols in the module scope and add
+  // their USE association to the current scope's USE-associated COMMON blocks.
+  for (const auto &[name, symbol] : useModuleScope_->commonBlocks()) {
+    if (!currScope().FindCommonBlockInVisibleScopes(name)) {
+      // Make a symbol, but don't add it to the Scope, since it needs to
+      // be added to the USE-associated COMMON blocks
+      Symbol *localCB{&currScope().MakeSymbol(
```

Comment:
This variable doesn't have to be a pointer.  `MakeSymbol` returns a reference, and you can declare `localCB` to be a reference that you can then just pass directly to `AddCommonBlockUse` as such.